### PR TITLE
📝 Add a guide on internet board games for the CC docs.

### PR DESCRIPTION
**Pull request by coderanger**
First pass from the google doc.

### CODE REVIEW

File: content/en/events/kcc2020/board-games.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+
+Just because we are internet-partying doesn’t mean we need to skimp on the board games! This is not an exhaustive list and if you have any questions please join us in the board-games channel on Discord!
+
+## Web
+
+All of these only require a web browser to get started.
+
+### Board Game Arena
+
+URL: https://boardgamearena.com/
```
Comment (castrojo) [inline content/en/events/kcc2020/board-games.md:16]: These aren't showing up as bullets/linefeeds in the markdown preview so they're all on the same line. Same for all the other games.

**Comment by mrbobbytables** [comment]
This is going back a bit on what I said in slack XD but after thinking about it more - I think it would probably be better to fold under our events and activities with board games as an H2 and then the rest under it as H3. 

What do you think?

**Comment by coderanger** [comment]
> This is going back a bit on what I said in slack XD but after thinking about it more - I think it would probably be better to fold under our events and activities with board games as an H2 and then the rest under it as H3.
> 
> What do you think?

Defer to you on IA, just worried that's getting to be a very long doc :)

**Comment by coderanger** [comment]
Reworked to be part of the activities page, along with reformatting to better support the new content.

**Comment by jeefy** [comment]
/lgtm
:taco:

**Pull request merged**